lib/platform-stats.js

```diff
@@ -199,3 +349,109 @@ export const updateTopMeasurementParticipants = async (createPgClient) => {
     Sentry.captureException(err)
   }
 }
+
+/**
+ * @param {import('pg').Client} pgClient
+ */
+export const aggregateAndCleanUpRecentData = async (pgClient) => {
+  try {
+    await pgClient.query('BEGIN')
+
+    await pgClient.query(`
+      WITH recent_station_details_summary AS (
+        SELECT
+          day,
+          SUM(accepted_measurement_count) AS accepted_measurement_count,
+          SUM(total_measurement_count) AS total_measurement_count,
+          COUNT(DISTINCT station_id) AS station_count,
+          COUNT(DISTINCT participant_id) AS participant_address_count
+        FROM
+          recent_station_details
+        WHERE
+          day <= CURRENT_DATE - INTERVAL '2 days'
```

Comment:
@PatrickNercessian @juliangruber do you happen to remember why we decided to use `INTERVAL '2 days'` here?

The consequence is that we don't have data for yesterday, e.g. to populate the "daily stations" chart.

If the intention was to ensure that `recent_station_details_summary` has complete data for the day being processed, then I think `INTERVAL '26 hours'` should be good enough.

Thoughts?

Reply:
I don't know tbh